<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebgisUtilityDialog</class>
 <widget class="QDialog" name="WebgisUtilityDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>686</width>
    <height>434</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="webgis_ip_edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>18</y>
     <width>791</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="person_id">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>152</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>154</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Регистрийн дугаар:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh_webgis_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>471</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh WebGIS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="find_ownership_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>177</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 127);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Find Ownership</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>201</y>
     <width>791</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Parcel information</string>
   </property>
   <widget class="QTableWidget" name="owner_twidget">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>14</y>
      <width>781</width>
      <height>181</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>180</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>19</number>
    </attribute>
    <column>
     <property name="text">
      <string>Parcel ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>first_name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>person_id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>aimag_name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>soum_name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>bag_name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>gudamj_name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>gudamj_dugaar</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>bair_dugaar</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>desion_date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>desion_number</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>area</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QPushButton" name="detail_print_button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>440</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Detail Pring</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>470</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 82, 48);</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>457</y>
     <width>801</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="open_webgis_button">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>433</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open WebGIS</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="webgis_url_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>434</y>
     <width>290</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>413</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>WebGIS Address</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>400</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>6</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>WebGIS Utility</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>7</y>
     <width>282</width>
     <height>13</height>
    </rect>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="check_connect_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Connect to WebGIS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>6</y>
     <width>134</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Find ownership progress:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="lm_rbutton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>85</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>LM</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="lpis_rbutton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>105</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>LPIS</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="subject_rbutton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>125</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Subject</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>421</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Information Detail</string>
   </property>
   <widget class="QTableWidget" name="owner_co_twidget">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>15</y>
      <width>408</width>
      <height>151</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>120</number>
    </attribute>
    <column>
     <property name="text">
      <string>Person ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Person Info</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Who</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QPushButton" name="defination_print_button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>410</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Defination Print</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/plugins/lm2/gzb.png</pixmap>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>person_id</tabstop>
  <tabstop>find_ownership_button</tabstop>
  <tabstop>open_webgis_button</tabstop>
  <tabstop>owner_twidget</tabstop>
  <tabstop>webgis_ip_edit</tabstop>
  <tabstop>refresh_webgis_button</tabstop>
  <tabstop>webgis_url_edit</tabstop>
  <tabstop>detail_print_button</tabstop>
  <tabstop>close_button</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
